react-components/src/components/RevealTopbar/RevealTopbar.tsx

```diff
@@ -24,6 +25,7 @@ const DefaultContentWrapper = (props: CustomTopbarContent): ReactElement => {
       <LayersButton />
       <SlicerButton />
       <FitModelsButton />
+      <RuleBasedOutputsContainer />
```

Comment:
We should expose this - further down in this file we expose a few other buttons, we can expose this likewise

Reply:
Ah ok.. Done here:
https://github.com/cognitedata/reveal/pull/4210/commits/e4c1637a2c6f4412f8fa79861d3d37844a646860